app/components/summary_card_component.html.erb

```diff
@@ -9,7 +9,13 @@
           </dt>
           <dd class="govuk-summary-list__value">
             <% if row[:value].is_a?(Array) %>
-              <% row[:value].each do |value| %><%= value %><br><% end %>
+              <% if @bullet == true %>
```

Comment:
:+1: to the feature — it would be good to have a spec too

Reply:
This will need a rebase on https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/574 (which also adds specs)

Reply:
For now I've rebased and moved the bullet list code out of the summary-card / list components here: https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/555/commits/bcabe35887beaaad7588f23cad391cb391c3cc41.

Currently this is the only page that requires a bulleted list in a summary card.
If this changes we can move functionality into the summary-list component.